client/test-data/bazaar/technopark/businesses/KücheEdison.json

```diff
@@ -0,0 +1,7 @@
+{
+	"name": "Küche Edison",
+	"description": "wir sind ein lokales Geschäft im Technopark und offerieren köstlichen Kaffe",
```

Comment:
this will be visible worldwide. I'd keep it informal "bei uns gibts köstlichen Kaffee". Shouldn't sound serious in any way